test/dataset.test.ts

```diff
@@ -15,22 +18,33 @@ DataLakeService.prototype.listFiles = jest
 
 DataLakeService.prototype.uploadFile = jest.fn();
 
-beforeAll(async () => {
-    await connectToDb(datasourceOptions);
-    const datafile1 = new Datafile();
-    datafile1.name = 'test-data-1.csv';
-    datafile1.description = 'Test Data File 1';
-    datafile1.id = 'bdc40218-af89-424b-b86e-d21710bc92f1';
-    await datafile1.save();
-    const datafile2 = new Datafile();
-    datafile2.name = 'test-data-2.csv';
-    datafile2.description = 'Test Data File 2';
-    datafile2.id = 'fa07be9d-3495-432d-8c1f-d0fc6daae359';
-    await datafile2.save();
-    console.log(`Datafile created: ${Datafile.find()}`);
-});
-
 describe('API Endpoints', () => {
+    beforeAll(async () => {
+        await connectToDb(datasourceOptions);
+        await dbManager.initializeDataSource();
+
+        const dataset1 = Dataset.createDataset('Test Data 1', 'test', 'bdc40218-af89-424b-b86e-d21710bc92f1');
+        dataset1.live = true;
+        dataset1.code = 'tst0001';
+        await dataset1.save();
+        dataset1.addTitleByString('Test Dataset 1', 'EN');
+        dataset1.addDescriptionByString('I am the first test dataset', 'EN');
```

Comment:
See above.  I believe this code longer term should live here as it will only be used here.  If I find I do need to use it in additional test suites then I'll move it into a helper which can be called.